prosody/Dockerfile

```diff
@@ -37,10 +40,16 @@ RUN wget -qO /etc/apt/trusted.gpg.d/prosody.gpg https://prosody.im/files/prosody
     apt-dpkg-wrap apt-get update && \
     apt-dpkg-wrap apt-get -d install -y jitsi-meet-prosody && \
     dpkg -x /var/cache/apt/archives/jitsi-meet-prosody*.deb /tmp/pkg && \
-    mv /tmp/pkg/usr/share/jitsi-meet/prosody-plugins /prosody-plugins && \
+    mv /tmp/pkg/usr/share/jitsi-meet/prosody-plugins $PROSODY_PLUGINS_FOLDER && \
     apt-cleanup && \
     rm -rf /tmp/pkg /var/cache/apt && \
-    patch -d /usr/lib/prosody/modules/muc -p0 < /prosody-plugins/muc_owner_allow_kick.patch
+    patch -d /usr/lib/prosody/modules/muc -p0 < $PROSODY_PLUGINS_FOLDER/muc_owner_allow_kick.patch
+
+RUN wget https://github.com/matrix-org/prosody-mod-auth-matrix-user-verification/archive/refs/tags/v$VERSION_MATRIX_USER_VERIFICATION_SERVICE_PLUGIN.tar.gz \
```

Comment:
You can use ADD to add the tarball and simplify the code here.

Reply:
Wouldn't this also add an additional layer?